src/build-scripts/build_openexr.bash

```diff
@@ -39,21 +47,43 @@ mkdir -p ${OPENEXR_INSTALL_DIR} && true
 pushd ${OPENEXR_SOURCE_DIR}
 git checkout ${OPENEXR_VERSION} --force
 
-cmake   -S . -B ${OPENEXR_BUILD_DIR} \
-        -DCMAKE_BUILD_TYPE=${OPENEXR_BUILD_TYPE} \
-        -DCMAKE_INSTALL_PREFIX="${OPENEXR_INSTALL_DIR}" \
-        -DCMAKE_PREFIX_PATH="${CMAKE_PREFIX_PATH}" \
-        -DILMBASE_PACKAGE_PREFIX="${OPENEXR_INSTALL_DIR}" \
-        -DOPENEXR_BUILD_UTILS=0 \
-        -DBUILD_TESTING=0 \
-        -DPYILMBASE_ENABLE=0 \
-        -DOPENEXR_VIEWERS_ENABLE=0 \
-        -DINSTALL_OPENEXR_EXAMPLES=0 \
-        -DOPENEXR_INSTALL_EXAMPLES=0 \
-        -DCMAKE_INSTALL_LIBDIR=lib \
-        -DCMAKE_CXX_FLAGS="${OPENEXR_CXX_FLAGS}" \
-        ${OPENEXR_CMAKE_FLAGS}
-time cmake --build ${OPENEXR_BUILD_DIR} --target install --config ${OPENEXR_BUILD_TYPE}
+if [[ ${OPENEXR_VERSION} == "v2.3.0" ]] ; then
+    # Simplified setup for 2.3+
+    cd ${OPENEXR_BUILD_DIR}
+    cmake -DCMAKE_BUILD_TYPE=${OPENEXR_BUILD_TYPE} \
+            -DBUILD_SHARED_LIBS=${OPENEXR_BUILD_SHARED_LIBS} \
+            -DCMAKE_INSTALL_PREFIX="${OPENEXR_INSTALL_DIR}" \
+            -DCMAKE_PREFIX_PATH="${CMAKE_PREFIX_PATH}" \
+            -DILMBASE_PACKAGE_PREFIX="${OPENEXR_INSTALL_DIR}" \
+            -DOPENEXR_BUILD_UTILS=0 \
+            -DOPENEXR_BUILD_TESTS=0 \
+            -DOPENEXR_BUILD_PYTHON_LIBS=0 \
+            -DCMAKE_CXX_FLAGS="${OPENEXR_CXX_FLAGS}" \
+            ${OPENEXR_CMAKE_FLAGS} ${OPENEXR_SOURCE_DIR}
+    time cmake --build . --target install --config ${OPENEXR_BUILD_TYPE}
+else
```

Comment:
I feel like maybe you forward-ported something from a while back. We recently simplified this code to no longer need the "v2.3.0" clause because OpenEXR 2.4 is now the oldest version we support.